themes/Valendas/lxqt-panel.qss

```diff
@@ -242,11 +242,14 @@ LXQtFancyMenuWindow {
     padding: 4px;
 }
 
-#FancyMenu QToolButton:hover {
+#FancyMenu QToolButton:hover,
+#FancyMenu QToolButton:focus {
     background: #444444;
     border: 1px solid grey;
 }
 
+#FancyMenu QToolButton:focus
```

Comment:
Oops! This seems redundant.

Reply:
Qt ignores empty blocks, but I'm not sure if it sees this as an empty block.

Reply:
I'm using that theme/PR, didn't notice anything all the time.
Should I fix it immediately?

Reply:
If you haven't noticed anything, then it's ignored and doesn't need to be fixed immediately.